modules/core/src/test/java/org/apache/ignite/internal/metric/IoStatisticsSelfTest.java

```diff
@@ -52,7 +52,7 @@
  */
 public class IoStatisticsSelfTest extends GridCommonAbstractTest {
     /** */
-    private static final int RECORD_COUNT = 5000;
+    private static final int RECORD_COUNT = 10000;
```

Comment:
Why this fix the test?

Reply:
in the PageMemoryImpl.acquirePage method (int grpId, long pageId, IoStatisticsHolder statHolder, boolean restore, @Nullable AtomicBoolean pageAllocated)

The page is loaded to the memory.

so we don't call IoStatisticsHolderCache.trackPhysicalAndLogicalRead

one of the solutions, to increase the amount of loaded data